### ISSUE TITLE
unpack_image returns string instead of array

### ISSUE THREAD

**Issue by ChrisBr** [SUSE/Enceladus#37]
It seems that https://github.com/SUSE/Enceladus/blame/master/ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py#L555 returns a long string for the studio tar files

```
Christians_SLES_12_JeOS_EC2-0.0.1/
Christians_SLES_12_JeOS_EC2-0.0.1/Christians_SLES_12_JeOS_EC2.x86_64-0.0.1.raw
Christians_SLES_12_JeOS_EC2-0.0.1/README.md
Christians_SLES_12_JeOS_EC2-0.0.1/create_ami.sh
print len(files) ==> 208
```

```
EC2_Upload-0.0.1/
EC2_Upload-0.0.1/README.md
EC2_Upload-0.0.1/create_ami.sh
EC2_Upload-0.0.1/EC2_Upload.x86_64-0.0.1.raw
```

So the script sometimes fails with `Unable to find raw image file with .raw extension`
because of the order of the files in the tar. My workaround for this is 
`files = files.split('\n')`

**Comment by ChrisBr** [comment]
Finally I get now:

```
Waiting for instance:  i-42
.  .  .  . 
Waiting for volume creation:  vol-42
. 
Wait for volume attachment

Waiting for volume creation:  vol-42
. 
Wait for volume attachment

Waiting to obtain instance public IP address

Attempt ssh connection
Formating storage volume
Creating ext3 filesystem on storage volume
Uploading image file:  42.tar.gz
. . . . . . . . .

Dumping raw image to new target root volume
Wait for volume to detach
. 
Waiting for snapshot creation:  snap-42
.  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  
Wait for volume to detach
.  .  .  .  . 
EC2ImageUploader instance has no attribute 'root_device_name'

```

**Comment by rjschwei** [comment]
Upload issue is fixed in version 0.5.0 as is the variable name issue.

**Comment by ChrisBr** [comment]
Many thanks for your help @rjschwei!! Just tried it with the new studio ec2 image and it uploads & boots fine :) I will test in detail next week!
